hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/systemcontracts/hts/create/CreateTranslator.java

```diff
@@ -152,7 +152,7 @@ public ClassicCreatesCall callFrom(@NonNull HtsCallAttempt attempt) {
                 attempt.systemContractGasCalculator(),
                 attempt.enhancement(),
                 nominalBodyFor(attempt),
-                attempt.defaultVerificationStrategy(),
+                attempt.contractVerificationStrategy(),
```

Comment:
Is this behaviour expected? Should we allow top level signatures for crypto create ?  

I am seeking opinions on this, since couldn't find how its exactly done it mono, because of the different flows.

Reply:
But wouldn't this result in the test failing in mono? 🤔